pandas/tests/plotting/test_frame.py

```diff
@@ -3316,6 +3316,22 @@ def test_missing_markers_legend_using_style(self):
         self._check_legend_labels(ax, labels=["A", "B", "C"])
         self._check_legend_marker(ax, expected_markers=[".", ".", "."])
 
+    def test_colors_of_columns_with_same_name(
```

Comment:
something like test_legend_colors_duplicate_labels ?

Reply:
Thanks for the reply, i have just made the changes . Can you check if is everything right now?